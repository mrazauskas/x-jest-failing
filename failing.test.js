test.failing("works with snapshots", () => {
  expect("two").toMatchSnapshot();
});
