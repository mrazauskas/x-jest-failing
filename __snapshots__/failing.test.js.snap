// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works with snapshots 1`] = `"one"`;
